prow/plank/controller.go

```diff
@@ -215,7 +219,7 @@ func (c *Controller) Sync() error {
 	}
 
 	var reportErrs []error
-	if c.ghc != nil {
+	if c.ghc != nil && !c.skipReport {
```

Comment:
this should probably just be c.report now?

Reply:
ghc is always not nil now... but that part can be changed? /shrug

/this-is-fine

Reply:
in that case we should have a sanity check and log a warning / error. if reporting is enabled, the client should exist?

Reply:
added the check inside the actual place references the github client